[Docs] More detailed info about args in Getting Started (#4058)
Cog-Creators/Red-DiscordBot: docs/getting_started.rst

    [p]help command

.. note::

>     Arguments enclosed in ``< >`` means that the argument is **required**
>     for the command to work.

Reviewer: It is easier to read sentences that are more concise:
```suggestion
    Arguments enclosed in ``< >`` are **required** for the command to work.
```

>     Arguments enclosed in ``[ ]`` means that the argument is **optional**
>     for the command; you can decide whether to use it or not.

Reviewer: It is easier to read sentences that are more concise:
```suggestion
    Arguments enclosed in ``[ ]`` are **optional** for the command;
    you can decide whether to use them or not.
```

    
    If your argument includes spaces like ``Hello world!``, most of the time
    you will need to place it in double quotes like this: ``"Hello world!"``.
    Sometimes (especially for the last argument) these double quotes are not
    required.

>     Arguments followed by ``=something`` means that, if not specified,
>     the argument will be equal to ``something``.

Reviewer: ```suggestion
    Arguments followed by ``=value`` means that, if not specified,
    the argument will be equal to ``value``.
```

  and enjoy Red!

  For that, just follow :ref:`this guide <systemd-service-guide>`.

.. _getting-started-userdocs:

------------------
User documentation
------------------

You will soon start using the Red core cogs. A detailed documentation is
available for every core cog, under the :ref:`How to use <main>` section.

The cog guides are formatted the same. They're divided into 3 sections:

* **Guide**

  This will introduce you to the cog and explain you how it works.

* **Commands**

  A list of the available commands, with details and arguments.
  Each command guide will be formatted like this:

  * **Syntax**

    A line that will show how the command must be invoked, with the arguments.

    .. tip:: If the command show something like ``[lavalinkset|llset]``, that means
        you can invoke the command with ``lavalinkset`` or with ``llset``, this is
        called an alias.

  * **Description**

    A detailed description of what the command does, with details about how
    it must be used.

  * **Arguments**

>     A list of all arguments needed (or not) for the command, with more details.

Reviewer: Cog guides are now a thing (thanks to Laggron ❤️), so this should no longer be removed.
